apps/src/aichat/views/assets/UploadButton.tsx

```diff
@@ -85,6 +97,15 @@ const UploadButton: React.FC<{isDisabled: boolean}> = ({isDisabled}) => {
         );
       }
     }
+
+    dispatch(
+      sendAnalytics(EVENTS.AICHAT_MULTIMODAL_UPLOAD_STAGED, {
+        source: AssetSource.PROJECT,
+        fileCountSuccess: uploadSuccessCount,
+        fileCountFailureSizeLimitExceeded: sizeLimitExceededCount,
+        fileCountFailureUnknownCause: uploadFailureCount,
```

Comment:
Nice! Do we also want to track if they tried to upload more than the file count limit (`stagedFilesLimitExceeded()` above)?

Reply:
I was trying to decide how much detail to include here, but seems like I landed on a maximalist approach 😆 . Added a count of the # of files that exceeded the max upload count as well.